src/superqComputation.cpp

```diff
@@ -493,11 +493,11 @@ void SuperqComputation::run()
 
         yDebug()<<"[SuperqComputation]: points size "<<points.size();
 
+        if (one_shot==false)
+            getPoints3D();
+
         if (points.size()>0)
         {
-            if (one_shot==false)
-                getPoints3D();
```

Comment:
`getPoint3D()` is in charge of filling the `points` container, thus it shouldn't be under the previous check `if (points.size()>0)`, apparently.